recipe/meta.yaml

```diff
@@ -1,14 +1,15 @@
-{% set version = "1.5.1" %}
+{% set version = "1.5.2" %}
 {% set numpy = "1.20.3" %}  # [py<310 and (not osx and arm64)]
-{% set numpy = "1.21" %}  # [py>=310 or (osx and arm64)]
+{% set numpy = "1.21" %}  # [py==310 or (osx and arm64)]
```

Comment:
Just to clarify, `osx-arm` is fine with numpy 1.21 on py311?

Reply:
It's not but it will be overridden by the next line.